src/decision/include/OdomToTwistStamped.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Created By: Cameron Newton
+ * Created On: Sept 12nd, 2018
+ * Description: A node that remaps the output of the decision node from an Odom
+ * message
+ *              to a TwistStamped for the simulator
+ */
+
+#ifndef JORMUNGANDR_ODOMTOTWISTSTAMPED_H
+#define JORMUNGANDR_ODOMTOTWISTSTAMPED_H
```

Comment:
This is just a general comment, but you can replace these with `#pragma once` (and get rid of the `#endif` at the end of the file) and it'll do the same thing. Not blocking this PR, but just so ya know (I only learned about this recently).

Reply:
Cool, I'll check that out in the future. TBH I don't really understand what these #ifndef markers mean